web/src/lib/components/admin-page/settings/notification-settings/notification-settings.svelte

```diff
@@ -93,6 +107,12 @@
               bind:value={config.notifications.smtp.from}
               isEdited={config.notifications.smtp.from !== savedConfig.notifications.smtp.from}
             />
+
+            <div class="w-1/2">
```

Comment:
It looks better on mobile without restricting the width to 50%, so I think the div can simply be removed.